pocs/observatory.py

```diff
@@ -591,7 +591,8 @@ def _create_mount(self, mount_info=None):
             mount_info = self.config.get('mount')
 
         model = mount_info.get('model')
-        port = mount_info.get('port')
+        serial_info = mount_info['serial']
```

Comment:
It does, it's part of the plan to support the dependency injection of the mount. All of the `_create_XXX` methods will be removed as part of that. I was tempted to start on that with this PR but it would have slowed it down a bunch more.

Reply:
That's fine, do it later.